### PR TITLE
Defficiency Medbay Machines

### PR DESCRIPTION
**Pull request by Kurfursten**
untitled
untitled

Also considering making a black labcoat for doctors who want it. Maybe a Spriter could come up with something cool for the CMO coat? As an optional thing in this case, for their lockers.

>Why is the mancrowave so small? Why no consoles for sleeper/FBS?

When this PR gets map priority I would like to refactor all three machines to be one tile entities making the console "built in" - it would reduce clunkiness that is caused by a lot of the two tile snowflakery. This requires editing every map though (to remove old consoles) and it could sit in review for a while so I don't want to get wrecked on conflicts.

🆑 
* tweak: Our medbay technology has undergone an upgrade. You no longer need a separate console for full body scanners, sleepers, or the mancrowave.
* rscadd: Defficiency now has a mancrowave and several unique black sprites (NanoMedPlus, Chemistry Smartfridge, Bloodbank Smartfridge, Roller Bed, Bioprinter/Prosthetics Fab, Sleeper, Mancrowave, Full Body Scanner). The CMO locker also now spawns a special extra suit slot item.

### CODE REVIEW

**Comment by ghost** [comment]
it's a MINICROWAVE aaaAAaAAAAAAAA THAT'S SO CUTE

**Comment by ShiftyRail** [comment]
More like Morgue Bay lmao

**Comment by Probe1** [comment]
gottem

**Comment by DeityLink** [comment]
Some of those (like the operating table) kinda looked better in white. Contrast is a nice thing you know?

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 1]
```diff
@@ -143,6 +143,11 @@
 	lockflags = DENSE_WHEN_LOCKED
 	lock_type = /datum/locking_category/buckle/bed/roller
 
+/obj/structure/bed/roller/New()
```
Comment (jknpj) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm]: What about the mediborg roller bed?

**Comment by jknpj** [comment]
Now all we need is 4 dirs sleepers

**Comment by Duny-** [comment]
Mancrowave is still a clutch for bads but at least this way it doesn't look like someone shat on their screen and called it a sprite.
The sleepers look a bit weird without any apparent console though.

**Comment by ShiftyRail** [comment]
In that configuration, where do the starting beakers of 30u cryox end up ?

**Comment by Kurfursten** [comment]
@ShiftyRail Its just a picture, the actual .dmm isn't edited. Probably chemistry though

@jknpj I'm not too worried about covering every use case, Medborgs themselves still have several skins that are prominently white, for example.

**Comment by Blithering** [comment]
sleeper
possibly something like this for the minisleepers?

**Comment by ghost** [comment]
For the refactor, since oldsleepers generate their console, if you changed the path to the minisleepers you'd simply have a bunch of clunky 1 tile gaps nobody cares about until you work your way through it all.

**Comment by jknpj** [comment]
@Kurfursten i mean won't it make their hover beds invisible?

**Comment by Kurfursten** [comment]
@Blithering I could incorporate the screen if you provide me your DMI

@SarahJohnson489 I plan on getting rid of the sleeper console object entirely so they need to be unmapped.

@jknpj That's a good point. For roller beds I'll need to do what I did originally and just make a deff subtype.

**Comment by Dacendeth** [comment]
Changing them to one tile will remove that weird bug with flatpacking those machines as well.

**Comment by Kurfursten** [comment]
I would like the Map Priority tag after #17466 clears

**Comment by Probe1** [comment]
Noted, will add it presumably tomorrow at the latest.

**Comment by Kurfursten** [comment]
Based on the feedback I think I will do away with the operating table sprite

Just gonna wait on that packed fix to clear before I rebase for the final push

**Comment by Kurfursten** [comment]
* OpTable sprite removed
* Consoles removed from all maps
* Tested and functional

**Comment by DamianX** [comment]
Is this good for the mergenatening?

**Comment by Probe1** [comment]
Give it till tomorrow evening so I can once over the maps for good measure to be extra safe and so no one gets mad.

**Comment by Probe1** [comment]
LETS GO

**Comment by Kurfursten** [comment]
Even after testing, editing 5 maps just makes you feel the anticipation of reports rolling in

**Comment by Probe1** [comment]
Such is life.

**Pull request merged**